src/io.c

```diff
@@ -28,14 +28,22 @@
 
 uint32_t G_output_len = 0;
 
+#ifdef HAVE_BAGL
 void io_seproxyhal_display(const bagl_element_t *element) {
     io_seproxyhal_display_default((bagl_element_t *) element);
 }
+#endif
 
 uint8_t io_event(uint8_t channel __attribute__((unused))) {
     switch (G_io_seproxyhal_spi_buffer[0]) {
+        case SEPROXYHAL_TAG_FINGER_EVENT:
+            UX_FINGER_EVENT(G_io_seproxyhal_spi_buffer);
```

Comment:
@fbeutin-ledger this could be between `#ifdef HAVE_NBGL`

Reply:
I've fixed this by taking the `io_event()` of boilerplate